csrankings-a.csv

```diff
@@ -919,6 +919,7 @@ Allison Bishop,Columbia University,http://www.cs.columbia.edu/~allison,NOSCHOLAR
 Allison Bishop Lewko,Columbia University,http://www.cs.columbia.edu/~allison,NOSCHOLARPAGE
 Allison K. Sullivan,University of Texas at Arlington,https://allisonius.github.io,omg10AMAAAAJ
 Allison Sullivan,University of Texas at Arlington,https://allisonius.github.io,omg10AMAAAAJ
+Allyson Ettinger,University of Chicago,https://linguistics.uchicago.edu/allyson-ettinger,tn6F5o0AAAAJ

```

Comment:
Linguistics, not CS. Please clarify how they meet the inclusion criteria.

Reply:
At the time of my previous pull request, Allyson's website explicitly said "apply to CS to work with me". Right now, it's not there or on the CS department (though Allyson can and does advise CS students and serves on our admissions committee) so I can remove if you need explicit documentation.

Reply:
Documentation would be ideal but if you ask her to attest here that she can solely advise PhD students _in Computer Science_, that would suffice for now. Thanks.

Reply:
Allyson declines due to some ambiguity in department policy. I've revised.